### PR TITLE
feat(typegraph-next): Add GraphQL runtime support

### PR DESCRIPTION
**Pull request by Natoandro**
Solves [MET-217](https://metatype.atlassian.net/browse/MET-217).

### CODE REVIEW

File: typegate/tests/e2e/typegraph/typegraphs/deno/graphql.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { g, t, typegraph } from "../../../../../../typegraph/deno/src/mod.ts";
+import { GraphQLRuntime } from "../../../../../../typegraph/deno/src/runtimes/graphql.ts";
+import * as effects from "../../../../../../typegraph/deno/src/effects.ts";
+
```
Comment (zifeo) [inline typegate/tests/e2e/typegraph/typegraphs/deno/graphql.ts:4]: you can put your test in the runtime/deno directly, I would keep e2e for more complex scenarios (like multiple pushes)

File: typegraph/deno/src/runtimes/graphql.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      runtime: this._id,
+      effect,
+    }, {
+      path,
+    });
+    return t.func(inp, out, new MutationMat(matId, effect, path));
+  }
+}
+
+export class QueryMat extends Materializer {
```
Comment (zifeo) [inline typegraph/deno/src/runtimes/graphql.ts:51]: interface/type instead?

File: typegate/tests/e2e/typegraph/typegraphs/deno/graphql.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { g, t, typegraph } from "../../../../../../typegraph/deno/src/mod.ts";
+import { GraphQLRuntime } from "../../../../../../typegraph/deno/src/runtimes/graphql.ts";
+import * as effects from "../../../../../../typegraph/deno/src/effects.ts";
```
Comment (zifeo) [inline typegate/tests/e2e/typegraph/typegraphs/deno/graphql.ts:3]: import_map

**Review by zifeo — APPROVED** [review]

**Review by afmika — APPROVED** [review]
Nice 👍

**Review by zifeo — DISMISSED** [review]

**Review by zifeo — APPROVED** [review]

**Pull request merged**